custom_components/youtube/sensor.py

```diff
@@ -123,6 +128,7 @@ def icon(self):
     def extra_state_attributes(self):
         """Attributes."""
         return {'url': self.url,
+                'friendly_name': self._name,
```

Comment:
There is no need to define this, HA adds that automatically

Reply:
I have removed friendly_name from the Attributes